packages/server/tests/clients/relayClient.ts

```diff
@@ -0,0 +1,114 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import { ethers, providers } from 'ethers';
+import { Logger } from 'pino';
+import Assertions from '../helpers/assertions';
+import { expect } from 'chai';
+
+export default class RelayClient {
+
+    private readonly provider: providers.JsonRpcProvider;
+    private readonly logger: Logger;
+
+    constructor(relayUrl: string, logger: Logger) {
+        this.logger = logger;
+        this.provider = new ethers.providers.JsonRpcProvider(relayUrl);
+    }
+
+    /**
+     * Calls the specified methodName with the provided params
+     * @param methodName
+     * @param params
+     */
+    async call(methodName: string, params: any[]) {
+        this.logger.debug(`[POST] to relay '${methodName}' with params [${params}]`);
+        const result = await this.provider.send(methodName, params);
+        this.logger.trace(`[POST] to relay '${methodName}' with params [${params}] returned ${JSON.stringify(result)}`);
+        return result;
```

Comment:
SHould we include expected checks here
```ts
expect(result).to.have.property('data');
        expect(result.data).to.have.property('id');
        expect(result.data).to.have.property('jsonrpc');
        expect(result.data.jsonrpc).to.be.equal('2.0');
```

Reply:
Actually moved my above suggestion into an `Assertions.successResponse()`

Reply:
Actually, the `provider` parses the response and returns only the actual information within `data`. Meaning if we console.log `result` we will get the underlying `data` and not the whole `json` object.